content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -188,3 +188,36 @@ To resolve this, we suggest you change your deployment to one with a replica set
 One simple way for this is to use [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) to launch a free instance that has replica set support out of the box.
 
 There's also an option to run the replica set locally with this guide: https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set
+
+## Type mapping between MongoDB and the Prisma schema
+
+> Alternatively, see the [Prisma schema reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types) for type mappings organized by Prisma type.
+
+The MongoDB connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) to MongoDB's native column types as follows:
+
+| Prisma    | MongoDB      |
+| --------- | ------------ |
+| `String`  | `String`     |
+| `Bool`    | `Boolean`    |
+| `Double`  | `Double`     |
+| `Int32`   | `Int32`      |
+| `Int64`   | `Int64`      |
+| `Decimal` | `Decimal128` |
+| `Date`    | `DateTime`   |
+| `BinData` | `Binary`     |
+
+When introspecting a MongoDB database, the database types are mapped to Prisma according to the following table:
+
+| MongoDB (Type \| Aliases) | Prisma     | Supported | Native database type attribute | Notes |
+| ------------------------- | ---------- | :-------: | :----------------------------- | :---- |
+| `ObjectId`                | `ObjectId` |    ✔️     | `@db.ObjectId`                 |       |
+
+[Introspection](/concepts/components/introspection) adds native database types that are **not yet supported** as [`Unsupported`](/reference/api-reference/prisma-schema-reference#unsupported) <span class="api"></span> fields:
+
+```prisma file=schema.prisma
+model Device {
+  id   Int                   @id @default(autoincrement())
+  name String
+  data Unsupported("circle")
```

Comment:
```suggestion
  data Unsupported("RegularExpression")
```
Not sure `Device` is a good example then any more though.

Reply:
I've updated it, though I don't have a good idea for an example model atm